Resources/Prototypes/Entities/Clothing/Eyes/hud.yml

```diff
@@ -16,15 +16,22 @@
   - type: ShowHealthIcons
 
 - type: entity
-  parent: ClothingEyesBase
+  parent: [ClothingEyesBase, BaseToggleClothing]
   id: ClothingEyesHudDiagnostic
   name: diagnostic hud
-  description: A heads-up display capable of analyzing the integrity and status of robotics and exosuits. Made out of see-borg-ium.
+  description: A heads-up display capable of analyzing the integrity and status of robotics and exosuits. Made out of see-borg-ium. Has a camera mod to work with Station AI.
   components:
   - type: Sprite
     sprite: Clothing/Eyes/Hud/diag.rsi
   - type: Clothing
     sprite: Clothing/Eyes/Hud/diag.rsi
+  - type: ToggleClothing
+    action: ActionToggleHudDiagnosticCamera
+    disableOnUnequip: true
+  - type: ComponentToggler
+    parent: true
+    components:
+    - type: StationAiOverlay
```

Comment:
This won't work because StationAiOverlay also (somehow) handles allowing access to items within view. 

In effect, this means you gain telekinesis because you can grab any item within view of your screen. This would need to be fixed, likely by separating the StationAiOverlay behavior into visual behavior and interaction behavior.